README.md

```diff
@@ -1,39 +1,156 @@
-# Project Template
+# Course Simulator
+The program is a course simulator where users can create and take different courses and get evaluated for them. We chose to create this program as it felt relevant as we are all students and the subject matter felt familiar!
```

Comment:
Would it be helpful to add an applications section or mention how the program could be used? For example, apart from being a course simulator, I think this program would be a pretty solid platform for quizzes. With some modifications or extensions, it could even be adapted for surveys, taking attendance, and other response-collecting tasks.

Reply:
That makes sense but I feel that is more geared toward presentation as well as the fact that application of this program is frankly up to the user so I'm not sure if it should be mapped out explicitly but I can include since it might be a nice addition to it.

Reply:
Yeah, just a mention is good. It doesn't need to be too detailed since technically it's just a suggestion for applications.